src/components/ActiveCard.jsx

```diff
@@ -26,18 +28,24 @@ class ActiveCard extends React.Component {
     };
   }
 
-  deactivateCard() {
-    this.props.dispatch(activateCard(null));
+  onDelete() {
```

Comment:
Have the new methods in this component crept in from a different PR? They don't seem to be related to translations.

Reply:
nvm. Are these existing methods that have been re-ordered by the linter?

Reply:
Yep, it's kinda irritating when the linter complains about method orders, but I make the switch to remove the error.